cypress/support/step-definitions/flash-steps.js

```diff
@@ -0,0 +1,32 @@
+import {
+  flashPreview, getIconPreview, messageInput, messagePreview, timeoutInput,
+} from '../../locators/flash';
+
+Then('Flash as is set to {string} and icon is set to {string}', (as, icon) => {
+  flashPreview().should('have.class', `carbon-flash--${as}`);
+  getIconPreview(icon).should('exist');
+});
+
+When('I set message property to {string}', (message) => {
+  messageInput().clear().type(message);
+});
+
+Then('Flash message is set to {string}', (message) => {
+  messagePreview().should('have.text', message);
+});
+
+When('I set timeout property to {int}', (timeout) => {
+  timeoutInput().clear().type(timeout);
+});
+
+When('I wait {int}', (timeout) => {
+  cy.wait(timeout);
+});
+
+Then('Flash is not visible', () => {
+  flashPreview().should('not.exist');
```

Comment:
exist or visible ?

Reply:
it is easier to read BDD - element is not visible instead of element not exists